src/components/PageContainer/PageContainer.tsx

```diff
@@ -26,14 +26,16 @@ export const PageContainer = ({ children }: PageContainerProps) => {
       <div className={classes.pageContainer}>
         <UserInfoBar />
```

Comment:
The page looks a little bit cramped with the UserInfo and the Exit button so close together imo. We probably don't need as much space between them as in the design, but I do think we need a bit more than this.
Maybe a gap of 30px or something?

![image](https://github.com/Altinn/altinn-access-management-frontend/assets/22012997/da2a9c97-0e2f-4594-ba7a-c4b605ab7a3e)